test_unstructured/staging/test_base_staging.py

```diff
@@ -102,6 +107,71 @@ def test_convert_to_dataframe_maintains_fields(
     assert "regex_metadata_punc" in df.columns
 
 
+def test_dataframe_dtypes():
```

Comment:
quick docstring on what you are testing..also maybe more specific test name? here it looks like you're specifically testing that for element there is a corollary default dtype?